### DISCUSSION TITLE
Uyuni salt minion error

### DISCUSSION THREAD

**Discussion by Alixis95** [uyuni-project/uyuni#7229] [Q&A]
Hi Experts,
I want to connect my uyuni server to my client via salt. I have the same versions of salt-master and salt-minion.  I have a single salt-master.

However, I get the following errors. Please help me to find the solution.

**Here is my salt-minion file (/etc/salt/minion.d/ uyuni.conf)**
master: host.name.com
grains:
  susemanager:
    activation_key: "key"
server_id_use_crc: adler32
enable_legacy_startup_events: False
enable_fqdns_grains: False

**salt-minion -l debug**
[DEBUG   ] Reading configuration from /etc/salt/minion
[DEBUG   ] Including configuration from '/etc/salt/minion.d/uyuni.conf'
[DEBUG   ] Reading configuration from /etc/salt/minion.d/uyuni.conf
[DEBUG   ] Using cached minion ID from /etc/salt/minion_id: host.name.com
[WARNING ] Insecure logging configuration detected! Sensitive data may be logged.
[DEBUG   ] Configuration file path: /etc/salt/minion
[INFO    ] Setting up the Salt Minion " host.name.com "
[INFO    ] An instance is already running. Exiting the Salt Minion
[INFO    ] Shutting down the Salt Minion
The Salt Minion is shutdown.

**/etc/salt/minion.d # systemctl status salt-minion**
● salt-minion.service - The Salt Minion
     Loaded: loaded (/usr/lib/systemd/system/salt-minion.service; enabled; vendor preset: disabled)
     Active: active (running) since Sun 2023-07-03 09:11:51 CEST; 14h ago
   Main PID: 33166 (salt-minion)
      Tasks: 3
     CGroup: /system.slice/salt-minion.service
             ├─31676 /usr/bin/python3 /usr/bin/salt-minion
             └─31681 /usr/bin/python3 /usr/bin/salt-minion

 [ERROR   ] Error while bringing up minion for multi-master. Is master at host.name.com >
Error while bringing up minion for multi-master. Is master at host.name.com >
salt-minion[ERROR   ][31681] Error while bringing up minion for multi-master. Is master at host.name.com  responding?
 salt-minion[ERROR   ][31686] Minion unable to successfully connect to a Salt Master.

Many Thanks
Best Regards

**Comment by digdilem-work** [comment]
Hi, 

My first thought is that there's something blocking the network route between the salt-minion and your master. 

Can the client ping the master?

Are all the ports opened as per the port list? https://www.uyuni-project.org/uyuni-docs/en/uyuni/installation-and-upgrade/ports.html

Does DNS on the client correctly resolve the master?

Is the certificate on the master trusted by the client? (I'd expect a SSL error here also if so, but throwing it in there)

**Comment by Alixis95** [comment]
Hello digdilem-work,

Many Thanks for you respons.
yes ping from client to master works.
How can I check DNS ?
I created a certificate on the master, but how can I trust the certificate by client ? 
Many Thanks
Best Regards
